packages/next-swc/crates/napi/npm/linux-arm64-gnu/install.js

```diff
@@ -0,0 +1,8 @@
+const { glibcVersionRuntime } = process.report.getReport().header;
+const { name } = require('./package.json');
+
+if (glibcVersionRuntime && name.endsWith('-gnu') || !glibcVersionRuntime && !name.endsWith('-gnu')) {
```

Comment:
```suggestion
if (glibcVersionRuntime) {
```
Should we keep the logic specific to each version so we don't need to read from `package.json`?

Reply:
Sounds good, I simplified it even more in d195de9